app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+const {
+  getNonexistantEndpoint,
+  getCategories,
+} = require("./controllers/index.js");
+
+app.get("/api/categories", getCategories);
+app.all("/*", getNonexistantEndpoint);
+
+const server = app.listen(9090, () => {
```

Comment:
We should extract our listen to a separate file, we don't need this for supertest as that creates it's own listener and in a test environment.

Reply:
My tests always seem to hang without saving this and exporting it to the test file to then close it in an afterAll(), is there something else I should be doing to prevent that instead?